### ISSUE TITLE
_comment.html form has a problem in timing.

### ISSUE THREAD

**Issue by thisiscetin** [beydogan/gitcussion#2]
``` ruby
<span class="date sub-text"><%= timeago_tag comment.created_at, :nojs => true, :limit => 100.days.ago %> </span> 
```

Even though I have commented 5 minutes ago, comment time shows as 'about 11 hours'. 
@ http://www.gitcussion.com/r/beydogan/gitcussion

**Comment by beydogan** [comment]
@manorie It might be about your time and timezone settings. Can you check your settings?

**Comment by thisiscetin** [comment]
It is fixed after you pushing another comment to top. Before any other comment present, I have refreshed the page in Safari OS X with correct time, locale settings but problem continued. It is strange, the problem is disappeared after your comment, gotta be a bug related to rails-timeago gem. You are not using some kind of caching at that page I think, strange.

**Comment by beydogan** [comment]
@manorie It was about timeago-tag in partial on new comments. Fixed with 2f355502b9817453615ab6bc3e5de2b50feebae8 
Thank you for letting me know.
